doc/user_guide/index.rst

```diff
@@ -38,32 +33,86 @@
    -----------------------------------------------------------------------------
    Written by R. W. Ford and A. R. Porter, STFC Daresbury Lab
 
+.. only:: html
+
+    .. image:: ../logo/psyclone_v1.0.png
+        :width: 75%
+        :align: center
+        :alt: PSyclone
+
+PSyclone User's Guide
```

Comment:
I've done some googling on "Users Guide", "User's Guide" etc. and think that "User Guide" is what we should call this document.

Reply:
Done